eng/pipelines/mgmt.yml

```diff
@@ -86,7 +86,6 @@ pr:
     - sdk/logic/Microsoft.Azure.Management.Logic
```

Comment:
I still suggest we use the https://github.com/Azure/azure-sdk-for-net/blob/master/eng/scripts/Update-Mgmt-Yml.ps1 to update this yml file so that it stays consistent.

Reply:
Thank you. Will do.